EnglishFriend/ViewModels/ProfileViewModel.swift

```diff
@@ -0,0 +1,19 @@
+ import Firebase
+ import SwiftUI
+
+ class ProfileViewModel: ObservableObject {
+
+     @Published var user: User
+
+     init (user: User)
+     {
+         self.user = user
+     }
+ }
+
+ extension ProfileViewModel {
+
+    //functions here
```

Comment:
Is this needed? If not let's remove comments so they don't wind up in prod someday.